specification/mcpp/resource-manager/Microsoft.CloudPartnerProgramMembership/preview/2023-02-09-preview/examples/getMcppPurchaseDetailsList.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "resourceGroupName": "McppPurchases",
+    "api-version": "2023-02-09-preview",
+    "subscriptionId": "291bba3f-e0a5-47bc-a099-3bdcb2a50a05"
+  },
+  "responses": {
+    "200": {
+      "body": {
+          "value": [ 
+            { 
+              "id": "microsoftactionpack",
+              "name": "microsoftactionpack",
+              "type": "string",
```

Comment:
type needs to be the RP name + resource type. In your case it would be something like:

```
"type": "Microsoft.CloudPartnerProgram/memberships"
```

Reply:
ack, will change this.